### DISCUSSION TITLE
Should react version be upgraded to 18 in Karavan-app?

### DISCUSSION THREAD

**Discussion by mrinalsharma** [apache/camel-karavan#617] [General]
Should react version be upgraded to 18 in Karavan-app?

**Comment by mgubaidullin** [comment]
Karavan's UI is based on [PatternFly](https://www.patternfly.org) and PatternFly just added [experimental](https://www.patternfly.org/v4/developer-resources/release-notes/) support for version 18 of React.

**Comment by mgubaidullin** [comment]
Need a lot of testing, especially for VS Code extension

**Comment by davsclaus** [comment]
closing old tickets
